operators/deploy/instance-operator/templates/deployment.yaml

```diff
@@ -47,6 +47,10 @@ spec:
             - "--container-env-vnc-img={{ .Values.configurations.containerEnvironmentOptions.vncImage }}"
             - "--container-env-websockify-img={{ .Values.configurations.containerEnvironmentOptions.websockifyImage }}"
             - "--container-env-novnc-img={{ .Values.configurations.containerEnvironmentOptions.novncImage }}"
+            - "--vm-registry={{ .Values.configurations.vmRegistry.url }}"
+            - "--vm-registry-secret={{ .Values.configurations.vmRegistry.secretName }}"
+            - "--container-export-img={{ .Values.configurations.imgExporter.imgName }}"
```

Comment:
This needs to be updated to leverage also the correct tag as [here](https://github.com/netgroup-polito/CrownLabs/blob/53ee64928941769aad33079baa2c2c8d928b2733/operators/deploy/instance-operator/templates/deployment.yaml#L46)